Resources/Prototypes/Entities/Mobs/Player/dragon.yml

```diff
@@ -159,3 +160,8 @@
     spawned:
     - id: FoodMeatDragon
       amount: 1
+  # half damage, spread evenly
+  - type: MeleeWeapon
+    damage:
+      groups:
+        Brute: 5
```

Comment:
This is supposed to be 15 and you added the component duplicated above.

Reply:
a

Reply:
i didnt duplicate

Reply:
Sire

![image](https://github.com/space-wizards/space-station-14/assets/31366439/f24b6c09-94d2-4e14-a994-5bfdaeb9d0e9)

Reply:
never happened your monitor must be broken